syncd/FlexCounter.cpp

```diff
@@ -1014,6 +1097,14 @@ FlexCounter::FlexCounter(
     m_enable = false;
     m_isDiscarded = false;
 
+    if (!sai_queue_api) {
+       sai_status_t status;
+       status = sai_api_query(SAI_API_QUEUE, (void **)&sai_queue_api);
```

Comment:
do not use gloal apis ! Vendor class should be used!

Reply:
Done